src/mixins/bcol-mixin.ts

```diff
@@ -8,7 +8,7 @@ export default class BcolMixin extends Vue {
       const fetchUrl: string = this.payApi + 'codes/errors/' + errCode
       // Currently no desirable way to handle errors during this request,
       // null is returned in any error situation regardless.
-      const errObj: any = await axios.get(fetchUrl).catch()
+      const errObj: any = await axios.get(fetchUrl)
```

Comment:
Have you tested the scenario where this get fails (exception)? Easy to do by corrupting the fetchUrl string above. This just to make sure the error is handled correctly.